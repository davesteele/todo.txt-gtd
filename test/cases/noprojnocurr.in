lonely task @computer

# _None
#

identified lonely task @computer
