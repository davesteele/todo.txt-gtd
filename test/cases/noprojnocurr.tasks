
@computer
lonely task @computer
identified lonely task @computer
