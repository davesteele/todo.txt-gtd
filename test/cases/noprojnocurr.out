# _None
#

identified lonely task @computer

lonely task @computer
